sdcm/mgmt.py

```diff
@@ -42,16 +43,17 @@ class HostSsl(Enum):
 
     @classmethod
     def from_str(cls, output_str):
-        try:
-            output_str = output_str.upper()
-            return getattr(cls, output_str)
-        except AttributeError:
-            raise ScyllaManagerError("Could not recognize returned host ssl status: {}".format(output_str))
+        if "SSL" in output_str:
+            return HostSsl.ON
+        return HostSsl.OFF
 
 
 class HostStatus(Enum):
     UP = "UP"
     DOWN = "DOWN"
+    TIMEOUT = "TIMEOUT"
+    UNAUTHORIZED = "UNAUTHORIZED"
+    HTTP = "HTTP"
```

Comment:
what is HTTP status? is it some new host status?

Reply:
I'm not sure, they didn't expand on it

Reply:
so please verify with them, as I don't see how the status of a host could be "HTTP"